shared/src/main/scala/coop/rchain/catscontrib/effect/implicits/package.scala

```diff
@@ -85,4 +85,39 @@ package object implicits {
         syncId.handleErrorWith(fa)(f)
       override def pure[A](x: A): Id[A] = x
     }
+
+  implicit val bracketTry: Bracket[Try, Throwable] = new Bracket[Try, Throwable] {
+
+    override def bracketCase[A, B](
+        acquire: Try[A]
+    )(use: A => Try[B])(release: (A, ExitCase[Throwable]) => Try[Unit]): Try[B] =
+      acquire.flatMap(
+        resource =>
+          cats.implicits.catsStdInstancesForTry
+            .attempt(use(resource))
+            .flatMap((result: Either[Throwable, B]) => {
+              val releaseEff =
+                result match {
+                  case Left(err) => release(resource, Error(err))
+                  case Right(_)  => release(resource, Completed)
+                }
+
+              cats.implicits.catsStdInstancesForTry.productR(releaseEff)(result.toTry)
+            })
+      )
+
+    override def raiseError[A](e: Throwable): Try[A] =
+      cats.implicits.catsStdInstancesForTry.raiseError(e)
+
+    override def handleErrorWith[A](fa: Try[A])(f: Throwable => Try[A]): Try[A] =
+      cats.implicits.catsStdInstancesForTry.handleErrorWith(fa)(f)
+
+    override def pure[A](x: A): Try[A] = cats.implicits.catsStdInstancesForTry.pure(x)
+
+    override def flatMap[A, B](fa: Try[A])(f: A => Try[B]): Try[B] =
+      cats.implicits.catsStdInstancesForTry.flatMap(fa)(f)
+
+    override def tailRecM[A, B](a: A)(f: A => Try[Either[A, B]]): Try[B] =
+      cats.implicits.catsStdInstancesForTry.tailRecM(a)(f)
```

Comment:
Do I understand that you can't import the `catsStdInstancesForTry` implicitly into a scope because it triggers the compilation errors? If so, then please asssign the `cats.implicits.catsStdInstancesForTry` to some local variable and call it, instead of `cats.implicits.catsStdInstancesForTry` every time.

Reply:
exactly. I had a lot of headaches with implicit conflicts when I imported all the implicits. By calling them explicitely I managed to get over the compilation errors. I would say it's not a big problem because the users of this class won't have these problems

Reply:
your suggestion to use a local name it's already there. See commit 2b74112